Cargo.lock

```diff
@@ -2846,7 +2847,6 @@ dependencies = [
  "der",
  "derive_more",
  "duct",
- "flume",
```

Comment:
notice that this doesn't bad flume from our codebase entirely...

Reply:
I am very much aware. There will be more PRs... I decided to do this one by one since some of them are more complex than others.

The biggest will be quic-rpc...